.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing
+So you want to contribute to this repository?
+
+When contributing to this repository, please first discuss the change you wish to make via issue with the owners of this repository before making a change.
```

Comment:
You use a very slight variation of "contribute to this repository" from the previous sentence.